src/AuthTokenFetcher.php

```diff
@@ -113,7 +114,7 @@ public function onBefore(BeforeEvent $event)
     //
     // TODO: correct caching; enable the cache to be cleared.
     $cached = $this->getCachedValue();
-    if (!is_null($cached)) {
+    if (!empty($cached)) {
```

Comment:
without this, I've run into empty strings being saved and then causing auth errors